src/Bitmap_cubical_complex/include/gudhi/Bitmap_cubical_complex.h

```diff
@@ -340,6 +340,7 @@ class Bitmap_cubical_complex : public T {
    * that provides ranges for the Boundary_simplex_iterator.
    **/
   Boundary_simplex_range boundary_simplex_range(Simplex_handle sh) { return this->get_boundary_of_a_cell(sh); }
+  Boundary_simplex_range coboundary_simplex_range(Simplex_handle sh) { return this->get_coboundary_of_a_cell(sh); }
```

Comment:
I think you need to add some comment for doxygen to document this function. Maybe add a Coboundary_simplex_range typedef? Or if you want to keep this private, why not call get_coboundary_of_a_cell directly in the file where you need it?

Reply:
I followed your suggestion and used `get_coboundary_of_a_cell` instead.